utils_linux.go

```diff
@@ -266,13 +266,17 @@ type runner struct {
 }
 
 func (r *runner) run(config *specs.Process) (int, error) {
```

Comment:
The correct way of handling this would've been to use `(_ int, Err error)` and then using the `error` in the defer. Otherwise, if you have a `return -1, fmt.Errorf()` you will treat it as a `nil`.